ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>Would it be worth adding an in-line explanation of what the setting is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added this!</p>